### ISSUE TITLE
Mirror function with a small detail

### ISSUE THREAD

**Issue by mbrakutt1** [AntonyCorbett/OnlyM#151]
Hello Anthony, is it possible to have the mirror function like Soundbox to display websites and images along with the browser toolbar you have built-in the web browser only on the mirror window? Just a thought, however I am open to suggestions 🙂

**Comment by AntonyCorbett** [comment]
@mbrakutt1 Perhaps. I was hoping the magnifier would be sufficient :)

**Comment by mbrakutt1** [comment]
A splendid feature indeed! Can you elaborate on your comment?

**Comment by AntonyCorbett** [comment]
@mbrakutt1 I hoped that the magnifier (and web page zoom) would allow the operator to see the media display in sufficient detail - allowing for navigation of a website - and so obviate the need for a mirrored window. Afterall, if you can't see the media in reasonable detail (albeit from the back of the hall), it's not much benefit to many in the audience.

**Comment by mbrakutt1** [comment]
@AntonyCorbett Makes sense I was looking at it from a very different viewpoint, I was thinking it might loose the audiences' attention if you have to for example zoom in with the magnifying tool. Simplicity was more on my mind with the audience just viewing the main output and the sound operator just needs to set his controls (necessary zoom level, magnifier etc.) and finally present.

**Comment by AntonyCorbett** [comment]
@mbrakutt1 it will be interesting to know if and how the magnifier is used.

**Comment by AntonyCorbett** [comment]
It would be possible to port the mirror window from SoundBox but not so easy to add the header panel only to the mirror.

**Comment by mbrakutt1** [comment]
> It would be possible to port the mirror window from SoundBox but not so easy to add the header panel only to the mirror.

I see, would a small window act as the main controls with the header on display 1 & and output unto the 2nd display a mirror? 🤔

But sizing might bring unexpected results?

**Comment by AntonyCorbett** [comment]
@mbrakutt1 In SoundBox, the mirror window is a reproduction of the original which is displayed on the media monitor. The canvas size of the mirror window matches the pixel resolution of the media monitor which means that the contents of the mirror window may need to be panned (which happens automatically as the mouse moves).

**Comment by mbrakutt1** [comment]
I see, what action will be taken from this point on wards?

**Comment by AntonyCorbett** [comment]
@mbrakutt1 I'll leave the issue open to see if anyone else wants to offer some input. I'll also evaluate feasibility of porting the SoundBox "mirror" code to OnlyM

**Comment by AntonyCorbett** [comment]
@mbrakutt1 the code has been ported from SoundBox so we can now display a "mirror" window when displaying websites and PDFs. Please see latest pre-release.

**Comment by mbrakutt1** [comment]
@AntonyCorbett Wow amazing thank you so much as always for your hard work, may Jehovah continue to shower you with his abundant blessings.
